src/churn/churn.ts

```diff
@@ -351,9 +350,15 @@ var log :logging.Log = new logging.Log('churn');
       //       'ciphertext_dfa': regex2dfa('^.*$'),
       //       'ciphertext_max_len': 1450
       //     }
-      this.pipe_.bindLocal(natEndpoints.internal);
-      this.pipe_.setBrowserEndpoint(webRtcEndpoint);
-      this.pipe_.bindRemote(remoteEndpoint).then(this.haveForwardingSocketEndpoint_);
+      this.pipe_.bindLocal(natEndpoints.internal).then(() => {
```

Comment:
I'd rather not slow down this process by serializing it.  It's fine to log something at completion but let's keep the three Promises starting in parallel.